src/site/content/en/progressive-web-apps/manifest-updates/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: How Chrome handles updates to the web app manifest
+subhead: What it takes to change icons, shortcuts, colors, and other metadata for your PWA
+authors:
+  - petelepage
+date: 2020-10-12
+updated: 2020-10-12
```

Comment:
```suggestion
date: 2020-10-14
updated: 2020-10-14
```

Reply:
I always want to make sure the thing's at the top of the list when it first hits the site. If you can get to this today, 10-13 is fine.